wbooks/src/app/screens/bookDetail/components/Comments/index.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types'; 
+import './style.css';
+
+class Comments extends Component {
+
+  constructor(props){
+    super();
+    this.state = {comment: ""};
+  }
```

Comment:
instead of overriding the constructor, define the state as `state = { comment: "" };`